### PR TITLE
Timeouts

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tihomir@temporal.io>

## What was changed:

Updated ActivityOptions to use StartToClose timeout

## Why?

same as in go examples -> https://github.com/temporalio/samples-go/pull/89
## Checklist

1. Closes issue: 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by tsurdilo** [comment]
@vitarb @Sushisource could you please review when you have a chance? Thanks!

File: src/main/java/io/temporal/samples/hello/HelloPolymorphicActivity.java [hunk 1]
```diff
@@ -119,20 +119,18 @@
      *
      * <p>For this example we define two activity stubs, one for each of the defined activities.
      *
-     * <p>Let's take a look at each {@link ActivityOptions} defined: The "setScheduleToCloseTimeout"
-     * option sets the overall timeout that the workflow is willing to wait for activity to
-     * complete. For this example it is set to 2 seconds for each of the defined activities.
+     * <p>Let's take a look at each {@link ActivityOptions} defined: The "setStartToCloseTimeout"
```
Comment (Sushisource) [inline src/main/java/io/temporal/samples/hello/HelloPolymorphicActivity.java:122]: This same wording `let's take a look at each {@link ActivityOptions} defined:` in all of these is a slightly awkward.

Perhaps instead

`In the {@link ActivityOptions} definition the "setStartToCloseTimeout"... `

**Review by Sushisource — APPROVED** [review]
Just a wording change suggestion

**Comment by tsurdilo** [comment]
@Sushisource thanks for the review! updated per suggestions.

**Pull request merged**